test/e2e/diagnose.go

```diff
@@ -43,22 +46,20 @@ func DiagnoseMe(clients *test.Clients, logger *logging.BaseLogger) {
 func checkCurrentPodCount(clients *test.Clients, logger *logging.BaseLogger) {
 	revs, err := clients.ServingClient.Revisions.List(metav1.ListOptions{})
 	if err != nil {
-		logger.Errorf(fmt.Sprintf("could not check current pod count: %v", err))
+		logger.Errorf("Could not check current pod count: %v", err)
 		return
 	}
 	for _, r := range revs.Items {
 		deploymentName := r.Name + "-deployment"
```

Comment:
While you are here, this can be replaced with`resourcenames.Deployment(r)`

Reply:
Done.